{{ cookiecutter.folder_name }}/pyproject.toml

```diff
@@ -34,7 +34,7 @@ name = "SLAC National Accelerator Laboratory"
 file = "LICENSE"
 
 [project.scripts]
-{{ cookiecutter.repo_name }} = "{{ cookiecutter.import_name }}.bin.main:main"
+"{{ cookiecutter.repo_name }}" = "{{ cookiecutter.import_name }}.bin.main:main"
```

Comment:
This is only here for migration tool compatibility. Without quotes, the variable in this line is technically invalid toml. 

It's still valid (and loads the same) with it quoted.